tests/conftest.py

```diff
@@ -120,6 +124,10 @@ def _configure_bigchaindb(request):
         test_db_name = '{}_{}'.format(TEST_DB_NAME, xdist_suffix)
 
     backend = request.config.getoption('--database-backend')
+
+    if backend == 'mongodb-ssl':
+        backend = 'mongodb'
+
     config = {
```

Comment:
Do we have all the information we need here to run the ssl related tests or do we still rely on env variables to be set?